sensor/sensorSender.py

```diff
@@ -18,79 +15,57 @@
-                    'sensor_id': f'{line}_PassageSensor{i + 1}',
-                    'type': 'passage_monitor',
-                    'cell_count': random.randint(1000, 5000)
-                }
-                channel.basic_publish(
-                    exchange='',
-                    routing_key='passage_monitor',
-                    body=json.dumps(sensor_data)
-                )
-                print('Sent passage_monitor data:', sensor_data)
-
-        time.sleep(5)  # Send data every 5 seconds
-
-
-# Sender function for fill_room_vat_monitor
-def send_fill_room_vat_monitor_data():
-    production_lines = ['Line1', 'Line2', 'Line3']
-    fill_room_sensors_per_line = 3
-
-    while True:
-        for line in production_lines:
-            for i in range(fill_room_sensors_per_line):
-                sensor_data = {
-                    'sensor_id': f'{line}_FillRoomSensor{i + 1}',
-                    'type': 'fill_room_vat_monitor',
-                    'room_temp': round(random.uniform(20, 25), 2),
-                    'humidity': round(random.uniform(40, 60), 2)
-                }
-                channel.basic_publish(
-                    exchange='',
-                    routing_key='fill_room_vat_monitor',
-                    body=json.dumps(sensor_data)
-                )
-                print('Sent fill_room_vat_monitor data:', sensor_data)
-
-        time.sleep(5)  # Send data every 5 seconds
-
-
-# Start sending data to respective queues
-send_flask_expansion_monitor_data()
-send_passage_monitor_data()
-send_fill_room_vat_monitor_data()
+def send_data(queue_name, data):
+    channel.basic_publish(exchange='', routing_key=queue_name, body=json.dumps(data))
+    print("Sent data to queue:", queue_name)
+    print(data)
+    print()
+
+
+# Generate sample data for flask_expansion_monitor
+def generate_flask_expansion_monitor_data():
+    data = {
+        'sensor_id': 'flask_expansion_sensor_1',
+        'type': 'flask_expansion_monitor',
+        'temp': 25.5,
+        'ph': 7.2,
+        'osmolality': 350
+    }
+    return data
+
+
+# Generate sample data for passage_monitor
+def generate_passage_monitor_data():
+    data = {
+        'sensor_id': 'passage_sensor_1',
+        'type': 'passage_monitor',
+        'cell_count': 5000
+    }
+    return data
+
+
+# Generate sample data for fill_room_vat_monitor
+def generate_fill_room_vat_monitor_data():
+    data = {
+        'sensor_id': 'fill_room_sensor_1',
+        'type': 'fill_room_vat_monitor',
+        'room_temp': 23.5,
+        'humidity': 60
+    }
+    return data
+
+
+# Send data every 5 seconds
+while True:
+    flask_expansion_data = generate_flask_expansion_monitor_data()
+    passage_data = generate_passage_monitor_data()
+    fill_room_data = generate_fill_room_vat_monitor_data()
+
+    send_data('flask_expansion_monitor', flask_expansion_data)
+    send_data('passage_monitor', passage_data)
+    send_data('fill_room_vat_monitor', fill_room_data)
+
```

Comment:
also i think this should be the same Queue name for each of these, we will use the type and sensor_id to read the data in correct place